prime-router/src/main/kotlin/azure/EmailEngineFunction.kt

```diff
@@ -307,7 +336,7 @@ class EmailScheduleEngine {
         template: String,
         emails: Iterable<String>,
         logger: Logger
-    ) {
+    ) : Response {
```

Comment:
What's the point of this returning a result? The call sites do not do anything with the result yet (cf line 150`dispatchToSendGrid(template, listOf(email), logger)`)
I see we log the warning here, is that enough and can we "get away" with not returning the response? We are doing logging here already, this may be enough... (or maybe this returns a `true`/`false` indicating success or failure)